acq4/devices/PMT/PMT.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from acq4.devices.Device import *
+from acq4.devices.OptomechDevice import OptomechDevice
+from acq4.devices.DAQGeneric import DAQGeneric
+
+class PMT(DAQGeneric, OptomechDevice):
```

Comment:
Let's go with `PhotoDetector` for the new class name and leave everything else as it is.